config.json

```
@@ -511,8 +511,8 @@
         "slug": "robot-name",
         "name": "Robot Name",
         "uuid": "46b8aea8-1528-43cb-a754-495ec2790ce2",
-        "practices": ["randomness", "classes"],
-        "prerequisites": ["randomness", "strings", "classes"],
+        "practices": ["randomness", "classes", "sets"],
+        "prerequisites": ["randomness", "strings", "classes", "sets"],
```

You don't necessarily _need_ to use sets to be able to solve this exercise (it can definitely help though). Maybe we add a prerequisite for `generic-types` instead (we don't yet have a `collections` concept`)?

```suggestion
        "prerequisites": ["randomness", "strings", "classes", "generic-types"],
```

---

If this is how indicate Callections, then yes sure. As I wrote we need required collections here

---

Yes, for now we only have `generic-types`. Having `collections` might be nice too later.